test/initialization-test/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This image has the script to kick off the ML pipeline initialization test,
 # and upload the result to GCS
 
-FROM golang:1.11
+FROM golang:1.13
```

Comment:
Why not `1.13.15`? It seems all others are using that version

Reply:
Sure, minor version was not defined before so left it, will update to 1.13.15